packages/mds-stream-processor/labelers/index.ts

```diff
@@ -0,0 +1,33 @@
+import db from '@mds-core/mds-db'
+import { UUID, VEHICLE_TYPE, PROPULSION_TYPE, Nullable, Telemetry } from '@mds-core/mds-types'
+
+type MessageLabeler<TMessage, TLabel> = (message: TMessage) => Promise<TLabel>
+
+export interface DeviceLabel {
+  vehicle_type: VEHICLE_TYPE
+  vehicle_propulsion: PROPULSION_TYPE[]
+}
+
+export const deviceLabeler: MessageLabeler<{ device_id: UUID }, DeviceLabel> = async ({ device_id }) => {
+  const { type: vehicle_type, propulsion: vehicle_propulsion } = await db.readDevice(device_id)
```

Comment:
Perhaps try hitting the cache first?

Reply:
This is a placeholder. I am thinking about using a small in-memory cache within the processor as opposed to `redis`, but yes, we should use some sort of caching here. Will be a follow on along with the point-in-polygon checks for geographies.

Reply:
Great, sounds good.